index.bs

```diff
@@ -12625,6 +12626,10 @@ internal method as follows.
     applied:
 
     1.  If <a abstract-op>Type</a>(|P|) is not String, then return <emu-val>false</emu-val>.
+    1.  If |P| is "<code>-0</code>", then return <emu-val>false</emu-val>.
+
+        Note: <a abstract-op>CanonicalNumericIndexString</a> allows "<code>-0</code>" as a special
+        case, even though it is not an [=integer index=] or by extension an [=array index=].
```

Comment:
If this indeed matches implementations, wouldn't it be cleaner to return false if index is -0 after the following step?

Reply:
The implementation for this in Chromium:
https://chromium.googlesource.com/v8/v8.git/+/842d424d99060375bf4a8b6fdfd2a0177e4c0477/src/api.cc#4128
https://chromium.googlesource.com/v8/v8.git/+/842d424d99060375bf4a8b6fdfd2a0177e4c0477/src/utils-inl.h#35

So no special handling of "-0" there because the code uses unsigned integers. ES uses ToNumber where negative zero is possible, so whatever this says won't be a close match.